x-pack/plugins/observability_ai_assistant/public/components/chat/function_list_popover.tsx

```diff
@@ -180,7 +180,7 @@ export function FunctionListPopover({
         }}
         singleSelection
         onChange={(options) => {
-          const selectedFunction = options.filter((fn) => fn.checked !== 'off');
+          const selectedFunction = options.filter((fn) => !('checked' in fn));
```

Comment:
`options` is an array, of which the item that the user has selected is the only item that does not have `checked`.